examples/ssh/ssl_certificate.json

```diff
@@ -0,0 +1,48 @@
+{
+    "logo": "ssl-checker.svg",
+    "version": "1.0.0",
+    "name": "SSL Certificate Expiration check",
+    "description": "Check whether a list of servers have valid SSL certificates and if there is any error with the Certificate Authority; check when the certificate expires; be ready to issue a new the Certificate by monitoring the remaining days on the current certificate",
+    "category": "Software Integration",
+    "requirements": {
+        "credentials": true,
+        "sandbox_version": "1.12",
+        "others": [
+            "Linux",
+            "openssl",
+            "ssl",
+            "tls"
+        ]
+    },
+    "tags": [
+        "linux",
+        "certificate",
+        "ssl",
+        "tls"
+    ],
+    "sample_period": 1800,
```

Comment:
change to sample_period_s

Maybe also add the execution_time_s to be 120 seconds (the max) so that we allow more servers to be monitored out of the box